packages/components/README.md

```diff
@@ -6,56 +6,73 @@ Reusable UI components for Deriv.
 
 **In this document**
 
--   [Other documents](#other-documents)
 -   [Commands](#commands)
 -   [Storybook](#storybook)
 -   [Usage](#usage)
 
-## Other documents
-
--   [General](docs/README.md) - Contains general philosophy and overview of this package
--   [Modules docs](docs/Modules/README.md) - Contains implementation guides (i.e., scaffolding, code usage)
-
 ## Commands
 
 Setup:
+The `npm run bootstrap` command should be run in the root of the project, because it does not exist in the components package.js
 
 ```sh
-lerna bootstrap
+npm run bootstrap
 ```
 
-Serve:
+Build:
+The `npm run build` command is used to build the components package
 
 ```sh
-npm run serve
+npm run build
 ```
 
-Build:
+Serve:
+The `npm run serve` command is used to serve the components package
 
 ```sh
-npm run build
+npm run serve components
 ```
 
 Test:
+The `npm run test` command is used to check whether the icon names are valid in the components package
 
 ```sh
 npm run test
 ```
 
+Test:
+The `npm run test:eslint` command is used to run eslint on the components package
```

Comment:
Maybe it would be good to mention that: we need to run this command inside component directory not in the root of the project
cc: @ali-hosseini-fs

Reply:
Yeah. I agree with you on this. I'll just add this emphasis

Reply:
Alright Jim, that's ok